wiki/Gameplay/Hit_object/Hit/en.md

```diff
@@ -0,0 +1,9 @@
+# Hit
+
+*Not to be confused with: [Katu judgement](/wiki/Gameplay/Judgement/Katu)*
+
+**Hits** are coloured circles present in [osu!taiko](/wiki/Game_mode/osu!taiko) which are called **Don** for the red variant and **Katsu** or **Katu** for the blue variant, respectively. **Finisher** are a bigger version of the regular hits which require the player to press two buttons of proper colour, instead of just one.
```

Comment:
```suggestion
**Don & Kat** are circles coloured in red and blue respectively which are present in [osu!taiko](/wiki/Game_mode/osu!taiko) [beatmaps](/wiki/Beatmap). **Finishers** (sometimes called *strong hits*) are a bigger version of the regular dons and kats that require the player to press two buttons of proper colour, instead of just one.
```